### PR TITLE
Analytics: Bug fixes and improved exception logging

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

File: octoprint_mrbeam/analytics/uploader.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def upload_file(self, my_file, token_data):
 				err = "upload_file failed: {}".format(r.status_code)
 				self.set_status(my_file, succ=False, err=err)
 		except Exception as e:
-			self._logger.exception("Exception while upload_file %s:", my_file)
+			self._logger.exception("Exception while upload_file {file}: {err}", file=my_file, err=e)
```
Comment (andywerner) [inline octoprint_mrbeam/analytics/uploader.py]: does this worK without the command format() ?

**Review by andywerner — CHANGES_REQUESTED** [review]
Please fix this one thing (or tell me why it works) and then merge it. thank you.

File: octoprint_mrbeam/analytics/uploader.py (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/analytics/uploader.py]: Oops. Fixed!

**Review by andywerner — APPROVED** [review]

**Comment by irlaec** [comment]
[Investigate errors from error list.](https://trello.com/c/vTB3ovUf/170-investigate-errors-from-error-list)

**Pull request merged**